bin/echo.c

```diff
@@ -81,8 +81,8 @@ int early_data_recv(struct s2n_connection *conn)
     ssize_t total_data_recv = 0;
     ssize_t data_recv = 0;
     bool server_success = 0;
-    s2n_blocked_status blocked = 0;
-    uint8_t *early_data_received = malloc(max_early_data_size);
+    s2n_blocked_status blocked = (s2n_blocked_status)0;
```

Comment:
What about using `S2N_NOT_BLOCKED` which is equal to 0? I see other tests just set it to 0 so there might be a reason for that.